actioncable/lib/action_cable/channel/base.rb

```diff
@@ -230,7 +230,7 @@ def subscribe_to_channel
 
 
         def extract_action(data)
-          (data['action'].presence || :receive).to_sym
+          data['action'].presence.to_sym || :receive
```

Comment:
Benchmarked it. Doesn't look like calling `to_sym` on a symbol is a potential bottleneck:

```ruby
require 'benchmark/ips'

Benchmark.ips do |x|
 x.report('to_sym') { :receive.to_sym }
end
```

Got:

```
Calculating -------------------------------------
              to_sym   131.252k i/100ms
-------------------------------------------------
              to_sym      7.973M (± 7.1%) i/s -     39.769M
```

Original seems fine, thanks! :grin: